lib/tasks/allrecipe-category-tags.rake

```diff
@@ -10,24 +10,32 @@ def get_category_info(main_category_name, main_category_url)
 
   @tries = 0
   begin
-    puts main_category_name
     puts main_category_url
     browser = open(main_category_url).read
     html_doc = Nokogiri::HTML(browser)
 
     sub_category_elements = html_doc.css('a#hlSubNavItem')
+    times_to_loop = html_doc.css('a#hlSubNavItem').size
+
+    if html_doc.css('ul#subNavGroupContainer.cat.related li') != nil
+      times_to_loop = times_to_loop - (html_doc.css('ul#subNavGroupContainer.cat.related li').size - 1)
```

Comment:
Should just remove items from an array and use `.each` function for the new array